templates/test-js.js

```diff
@@ -0,0 +1,59 @@
+module.exports = answers => {
+	const name = {
+		original: answers.name,
+		titleCase: answers.name.split('-').map(word => word.replace(word[0], word[0].toUpperCase())).join('')
```

Comment:
This function is defined in two files, perhaps we should move it into it's own file/module and give it a nice name for what it is doing :-)